gems/quilt_rails/lib/quilt_rails/react_renderable.rb

```diff
@@ -8,11 +8,16 @@ def render_react
       url = "#{Quilt.configuration.react_server_protocol}://#{Quilt.configuration.react_server_host}"
       ReactRenderable.log("[ReactRenderable] proxying to React server at #{url}")
 
-      reverse_proxy(url) do |callbacks|
-        callbacks.on_response do |status_code, _response|
-          ReactRenderable.log("[ReactRenderable] #{url} returned #{status_code}")
+      begin
+        reverse_proxy(url) do |callbacks|
+          callbacks.on_response do |status_code, _response|
+            ReactRenderable.log("[ReactRenderable] #{url} returned #{status_code}")
+          end
         end
+      rescue Errno::ECONNREFUSED => e
```

Comment:
Was looking into adding a test for this. How do I mock a`Errno::ECONNREFUSED` in Rails,  any examples on hand?

Reply:
`Errno::ECONNREFUSED.new` gives an instance of it in `irb`

Reply:
so I would just mock `reverse_proxy` to throw one

Reply:
Opened an issue to get tests working
https://github.com/Shopify/quilt/issues/883